app/app_test.go

```diff
@@ -113,6 +113,7 @@ func TestExportAppStateAndValidators(t *testing.T) {
 			MarkerType:             markertypes.MarkerType_RestrictedCoin,
 			SupplyFixed:            true,
 			AllowGovernanceControl: true,
+			ClawbackEnabled:        false,
```

Comment:
We should consider using the term "ForcedTransfer" as a clawback is a specific type of forced transfer but this implementation supports more than just pulling the tokens back into the marker account.